public/kibana-integrations/kibana-vis.js

```diff
@@ -371,6 +371,10 @@ class KibanaVis extends Component {
       const currentCompleted = Math.round((currentLoaded / totalTabVis) * 100);
       if (currentCompleted >= 100) {
         this.$rootScope.rendered = 'true';
+        if(this.visID === 'Wazuh-App-Overview-General-Agents-status'){
+          this.$rootScope.resultState = 'none';
+          this.tabVisualizations.addDeadVis();
+        }
```

Comment:
**thought**: Why is it required?

Reply:
I don't know why. But I'd prefer don't touch the `kibana-vis` component.